Content.Server/NPC/HTN/PrimitiveTasks/Operators/Specific/PickNearbyInjectableOperator.cs

```diff
@@ -53,10 +57,16 @@ public override void Initialize(IEntitySystemManager sysManager)
                 damage.TotalDamage > 0 &&
                 !recentlyInjected.HasComponent(entity))
             {
+                var path = await _pathfinding.GetPath(owner, entity, SharedInteractionSystem.InteractionRange, cancelToken);
+
+                if (path.Result == PathResult.NoPath)
+                    continue;
+
                 return (true, new Dictionary<string, object>()
                 {
                     {TargetKey, entity},
-                    {TargetMoveKey, _entManager.GetComponent<TransformComponent>(entity).Coordinates}
+                    {TargetMoveKey, _entManager.GetComponent<TransformComponent>(entity).Coordinates},
+                    {"MovementPathfind", path},
```

Comment:
shouldnt this be a const somewhere (like `NPCBlackboard.cs`, not sure if those consts refer to the same type of thing here)

Reply:
sur